ontology/docs/hvac_chws.md

```diff
@@ -0,0 +1,329 @@
+        values:
+          percent: '%'
+    ...
+
+AHU-1:
+  connections:
+    ZONE-2: FEEDS
+    BLDG-1: CONTAINS
+  type: HVAC/AHU_DFSS_...
+  translation:
+    chilled_water_valve_percentage_command:
+      present_value: points.chwv.present_value
+      units:
+        key: pointset.points.chwv.units
+        values:
+          percent: '%'
+    ...
+
+
+```
+
+## Example: Variable Primary CHWS (Water-Cooled) 
+This version of CHWS has a single set of variable speed chilled water pumps serving water-cooled chillers and downstream equipment. The condensing side serves the chillers via two pumps.
+
+### BMS Example
+![CHWS Variable Primary WC](./figures/bms_screenshots/chws_watercooled.png)
+**Notes:**
+- This example includes a parallel air-cooled chiller. It is simple enough to add; it doesnt change the way the rest of the system is modeled.
+
+### System Diagram and Connections
+![CHWS Variable Primary WC](./figures/system_diagrams/chws_watercooled.png)
+
+### Sample Building Config
+```yaml
+CHWSYS-BLDG-1:
+  connections:
+    CDWSYS-BLDG-1: FEEDS
+  type: HVAC/HWS_...
+  translation:
+    supply_water_temperature_sensor:
+      present_value: points.sys_swt.present_value
+      units:
+        key: pointset.points.sys_swt.units
+        values:
+          degrees_fahrenheit: 'deg-F'
+    differential_pressure_sensor:
+      present_value: points.sys_dp.present_value
+      units:
+        key: pointset.points.sys_dp.units
+        values:
+          pascals: 'Pa'
+    ...
+
+CDWSYS-BLDG-1:
+  connections:
+    ...
+  type: HVAC/CDWS_...
+  translation:
+    supply_water_temperature_sensor:
+      present_value: points.sys_swt.present_value
+      units:
+        key: pointset.points.sys_swt.units
+        values:
+          degrees_fahrenheit: 'deg-F'
+    ...
+
+CT-1:
+  connections:
+    CHWSYS-BLDG-1: CONTAINS
+  type: HVAC/CT_...
+  translation:
+    supply_water_temperature_sensor:
+      present_value: points.swt.present_value
+      units:
+        key: pointset.points.swt.units
+        values:
+          degrees_fahrenheit: 'deg-F'
+    ...
+
+CT-2:
+  connections:
+    CHWSYS-BLDG-1: CONTAINS
+  type: HVAC/CT_...
+  translation:
+    supply_water_temperature_sensor:
+      present_value: points.swt.present_value
+      units:
+        key: pointset.points.swt.units
+        values:
+          degrees_fahrenheit: 'deg-F'
+    ...
+
+CH-1:
+  connections:
+    CHWSYS-BLDG-1: CONTAINS
+  type: HVAC/CHL_...
+  translation:
+    supply_water_temperature_sensor:
+      present_value: points.swt.present_value
+      units:
+        key: pointset.points.swt.units
+        values:
+          degrees_fahrenheit: 'deg-F'
+    ...
+
+CH-2:
+  connections:
+    CHWSYS-BLDG-1: CONTAINS
+  type: HVAC/CHL_...
+  translation:
+    supply_water_temperature_sensor:
+      present_value: points.swt.present_value
+      units:
+        key: pointset.points.swt.units
+        values:
+          degrees_fahrenheit: 'deg-F'
+    ...
+
+ACC-1:
```

Comment:
"ACC" does not align to a BDNS or BOS abbreviation
https://github.com/theodi/BDNS/blob/master/BDNS_Abbreviations_Register.csv

Reply:
for brownfield sites this will be whatever the underlying asset name is (and the example represents a brownfield site). I will add a note saying as much.